tests/core_tests.py

```diff
@@ -155,8 +156,9 @@ def test_slices_V2(self):
                 (slc.slice_name, 'slice_url', slc.slice_url),
             ]
         for name, method, url in urls:
-            print("[{name}]/[{method}]: {url}".format(**locals()))
+            logging.info("[{name}]/[{method}]: {url}".format(**locals()))
```

Comment:
logging is more flexible and it possible to choose what log severity you care about.
http://stackoverflow.com/questions/6918493/in-python-why-use-logging-instead-of-print